packages/docsearch-react/src/NoResultsScreen.tsx

```diff
@@ -68,17 +68,18 @@ export function NoResultsScreen({
         </div>
       )}
 
-      <p className="DocSearch-Help">
-        {`${openIssueText} `}
-        <a
-          href={`https://github.com/algolia/docsearch-configs/issues/new?template=Missing_results.md&title=[${props.indexName}]+Missing+results+for+query+"${props.state.query}"`}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          {openIssueLinkText}
-        </a>
-        .
-      </p>
+      {props.reportMissingResultsUrl && (
+        <p className="DocSearch-Help">
+          {`${reportMissingResultsText} `}
+          <a
+            href={props.reportMissingResultsUrl}
```

Comment:
Why don't we pass the query to a function? It [seems to be widely used](https://github.com/algolia/docsearch-configs/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+missing+results) and makes the user request accurate. We're loosing information with this update.

Reply:
Do you mean what is suggested in the `Other solutions`?

We could definitely have `reportMissingResultsUrl` as a function `(query:string) => string` so the user can build the appropriate link.

Reply:
`({ query }) => string`, yes.

Reply:
works as expected: https://github.com/algolia/docsearch/pull/1289/commits/09346644629173af0287c8b4f1b7c15254e1bb0d#diff-f2decf6ff1d6fc42353fede839f1280094117430c1253a0359a54ea725d36a3aR256-R258

Reply:
Not sure about the name 🤔